tools/release.sh

```diff
@@ -45,7 +45,7 @@ export GPG_TTY=$(tty)
 PORT=${PORT:-1234}
 
 # subpackages to be released
-SUBPKGS=${SUBPKGS:-"acme certbot-apache certbot-nginx letshelp-certbot"}
+SUBPKGS=${SUBPKGS:-"acme certbot-apache certbot-nginx letshelp-certbot letsencrypt letsencrypt-apache letsencrypt-nginx letshelp-letsencrypt"}
```

Comment:
I think so, at least for the next release. The package is currently useless, however, we have been packaging it every release and we want to make sure people are able to upgrade if the package ever becomes useful.

With that said, after the next release, perhaps we should stop packaging `certbot-nginx` and `letshelp-certbot` until the packages are ready.